src/main/java/io/druid/data/input/InputRow.java

```diff
@@ -16,7 +17,7 @@
  * in all lowercase form (i.e. they should either have already turned everything into lowercase or they
  * should operate in a case-insensitive manner).
  */
-public interface InputRow extends Row
+public interface InputRow extends Row ,Serializable
```

Comment:
Also why Serializable instead of allowing Jackson serde would be a nice thing to know.